docs/cni/kubernetes/GCE.md

```diff
@@ -73,7 +73,7 @@ gcloud compute instances create \
   --image-project coreos-cloud \
   --image coreos-stable-835-9-0-v20151208 \
   --machine-type n1-standard-1 \
-  --metadata-from-file user-data=cloud-config/master-config.yaml
+  --metadata-from-file user-data=cloud-config/master-config-ipip.yaml
```

Comment:
Yes: all the other references to this file; they just don't show up in the diff because they didn't need changing.